src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<blockquote>criticism</blockquote><p>suggestions </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Changed in dfe233f7

Reply:
<p>Variable viewer behaves poorly because variable names have no colon. Maybe we should enhance it...</p><p> </p>

Reply:
On one hand I agree that we should enhance the looks, on the other hand I like the fact that names are consistent compared to the pure python implementation.

But if I were to do it, the inputs are already sorted (geometry/target mission/propulsion technology). However, since the there is only one item in propulsion technology, wouldn't it behave bad as well ?

Reply:
<p>I realize there is a slight misunderstanding here, that I am not sure we cleared up when we discussed (I had not seen your reply then). My point was on the variable viewer itself (hence Scott's work in PR#45).</p>

Reply:
We did, just after I posted the comment.